cmd/up/up.go

```diff
@@ -613,14 +613,18 @@ func (up *upContext) deployApp(ctx context.Context, ioCtrl *io.Controller, k8slo
 		isRemote = up.Manifest.Deploy.Image != ""
 	}
 
+	// We keep DeprecatedOktetoCurrentDeployBelongsToPreview for backward compatibility in case an old version of the backend
+	// is being used
+	isPreview := os.Getenv(model.DeprecatedOktetoCurrentDeployBelongsToPreview) == "true" ||
+		os.Getenv(constants.OktetoIsPreviewEnvVar) == "true"
```

Comment:
This is not related to the destroy refactor per se, it is related with [this ticket](https://okteto.atlassian.net/browse/DEV-245)